frontend/src/static/css/details-screen.scss

```diff
@@ -0,0 +1,151 @@
+@import './variables.scss';
+
+.container-main {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 90%;
+  width: 100%;
+  margin: auto 0;
+
+  .container-graphic {
+    display: flex;
+    flex-direction: column;
+    width: 50%;
+    height: 100%;
+    background: $main-color;
+    box-shadow: 0px 4px 30px rgba(157, 157, 157, 0.25);
+    border-radius: 14px 0px 0px 14px;
+
+    .title-graphic {
+      color: #fff;
+      font-size: 20px;
+      margin: 5% auto;
+    }
+
+    .container-graphic-pie {
+      display: flex;
+      height: 100%;
+
+      .graphic-pie {
+        display: flex;
+        align-items: center;
+        flex-direction: column;
+        width: 50%;
+        margin: auto 1%;
+  
+        p {
+          color: #fff;
+          font-size: 17px;
+          margin-bottom: 7%;
+        }
+      }
+    }
+    
+  }
+
+  .container-product {
+    display: flex;
+    flex-direction: column;
+    width: 47%;
+    height: 100%;
+    background: #fff;
+    box-shadow: 0px 4px 30px rgba(157, 157, 157, 0.25);
+    border-radius: 0px 14px 14px 0px;
+
+    .product {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      height: 30%;
+      width: 100%;
+
+      .photo-product {
+        width: 50%;
+        height: 90%;
+      }
+
+      .title {
+        color: #515151;
+        font-size: 18px;
+        font-weight: 700;
+      }
+    }
+
+    .content-product {
+      display: flex;
+      margin: 2% 5%;
+      justify-content: space-between;
+
+      div .price {
+        color: $main-color;
+        font-size: 20px;
+      }
+
+      .info-platform {
+        cursor: pointer;
+        display: flex;
+        justify-content: center;
+        width: 30%;
+        height: 30px;
+        align-items: center;
+        border-radius: 24px;
+        background: #f9a94b;
+
+        .bag-icon {
+          margin-right: 5%;
+        }
+
+        p {
+          color: #7d490d;
+        }
+      }
+    }
+
+    .description {
+      width: 90%;
+      margin-left: 5%;
+      margin-bottom: 1%;
+      color: #949494;
+    }
+
+    .feedback {
+      height: 100%;
```

Comment:
I'm thinking only about you use the height property in almost all containers, i need verify, but for now it's ok